### ISSUE TITLE
SquirrelSQL: Node / Relationship - Results don't work, all columns null

### ISSUE THREAD

**Issue by jexp** [neo4j/neo4j-jdbc#32]
```
:play movies

MATCH (actor:Person {name: "Tom Hanks"})-[r:ACTED_IN]->(movie:Movie) RETURN actor,r,movie
```

<img width="1078" alt="squirrel sql client version 3 7 1" src="https://cloud.githubusercontent.com/assets/67427/15894227/00648bca-2d85-11e6-8f9c-e3a528c14e07.png">

**Comment by sim51** [comment]
Related to #43 & #48

**Comment by glaggia-larus** [comment]
commit 260a5d3  adds two failing tests that highlight the problem

**Comment by albertodeste** [comment]
Fixed in 3e34ea8
